yaml/xyz/openbmc_project/State/Decorator/PowerState.interface.yaml

```diff
@@ -12,10 +12,10 @@ enumerations:
     description: >
       Power state enum.
     values:
-      - name: On
+      - name: "On"
```

Comment:
Why change this?

Reply:
This is from https://gerrit.openbmc-project.xyz/c/openbmc/phosphor-dbus-interfaces/+/46792. Needed to fix the CI errors